server/handlers/user_handler.go

```diff
@@ -193,3 +193,23 @@ func (h *Handler) ShareDesignHandler(w http.ResponseWriter, r *http.Request, _ *
 	w.Header().Set("Content-Type", "application/json")
 	fmt.Fprint(w, "Design shared")
 }
+
+// swagger:route POST /api/content/filter/share ShareContent idPostShareContent
+// Handle POST request for Sharing content
+//
+// Used to share filters with others
+// responses:
+// 	200:
+//  403:
+//  500:
+
+func (h *Handler) ShareFilterHandler(w http.ResponseWriter, r *http.Request, _ *models.Preference, _ *models.User, provider models.Provider) {
+	statusCode, err := provider.ShareDesign(r)
```

Comment:
```suggestion
	statusCode, err := provider.ShareFilter(r)
```
It will be shareFilter right?